### ISSUE TITLE
Double-click on *.svg file should show image, not show source

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#1191]
Right now if user double-clicks on a *.svg file, we show the SVG source code. Instead, default action should be to show the visual result like we do for *.png, *.gif and *.jpg.

**Comment by peller** [comment]
it would be neat to have a split editor!

**Comment by jhpedemonte** [comment]
Yes, came here to say what Adam said. Maybe we should handle SVG similar to HTML -- show visual content by default but allow user to view and edit source.  Although there might be issues with how source and visual views work together, issues we've already resolved for HTML but not for other viewers.

**Comment by JonFerraiolo** [comment]
Yes, it seems like it wouldn't be too hard to support SVG authoring in split view where the source view is editable but the visual view is view-only (versus HTML where the visual view is editable). (Personally, I would find it to be a really awesome feature!)

**Comment by JonFerraiolo** [comment]
@waynevicknair should be aware of this
